docs/reference/announcements/870.md

```diff
@@ -10,7 +10,7 @@ Important changes and updates for the Camunda 8.7 release are summarized below.
 
 | Scheduled release date | Scheduled end of maintenance | Release notes                                        | Blog                                                                                            |
 | :--------------------- | :--------------------------- | :--------------------------------------------------- | :---------------------------------------------------------------------------------------------- |
-| 11 February 2025       | 11 August 2026               | [8.7 release notes](/reference/release-notes/870.md) | [Announcing Camunda 8.7](https://camunda.com/blog/2024/11/camunda-8-7-releasing-february-2025/) |
+| 8 April 2025           | 6 October 2026               | [8.7 release notes](/reference/release-notes/870.md) | [Announcing Camunda 8.7](https://camunda.com/blog/2024/11/camunda-8-7-releasing-february-2025/) |
```

Comment:
Nit-pick: would we not have end of maintenance on the second Tuesday of October 2026 OR maybe on 6 October?

Reply:
Thank you, I went the wrong way on the calendar. 😅

Reply:
Oct 13 looks better, right?